android/src/main/java/com/robinpowered/RNAudioJack/RNAudioJackModule.java

```diff
@@ -2,20 +2,83 @@
 
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.facebook.react.bridge.ReactMethod;
-import com.facebook.react.bridge.Callback;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.WritableNativeMap;
 
-public class RNAudioJackModule extends ReactContextBaseJavaModule {
+import java.util.Map;
+import java.util.HashMap;
+import javax.annotation.Nullable;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Build;
+import android.media.AudioDeviceInfo;
+import android.media.AudioManager;
 
-  private final ReactApplicationContext reactContext;
+public class RNAudioJackModule extends ReactContextBaseJavaModule {
+  private static final String MODULE_NAME = "RNAudioJack";
+  private static final String AUDIO_CHANGED_NOTIFICATION = "AUDIO_CHANGED_NOTIFICATION";
+  private static final String IS_AUDIO_JACK_PLUGGED_IN = "isAudioJackPluggedIn";
```

Comment:
Echoing @christophermark's feedback in https://github.com/robinpowered/react-native-audio-jack/pull/2/files#r159991719 to do the same here, `isPluggedIn`